helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -3,48 +3,59 @@ kind: PrometheusRule
 metadata:
   labels:
     {{- include "labels.common" . | nindent 4 }}
-    cluster_type: "control_plane"
+    cluster_type: "management_cluster"
```

Comment:
splitting up PRs between re-enabling cortex and updating labels would be helpful (even though i hate writing that)

Reply:
Splitted with https://github.com/giantswarm/prometheus-meta-operator/pull/285